modules/config/variables.tf

```
@@ -0,0 +1,45 @@
+variable "name" {
+  description = "Name to set on AWS Config resources."
+  type        = string
+  default     = "default"
+  nullable    = false
+}
+
+variable "bucket" {
+  type        = string
+  description = "The name of the S3 bucket used to store the configuration history."
+  nullable    = false
+}
+
+variable "prefix" {
+  description = "The prefix for the specified S3 bucket."
+  type        = string
+  default     = ""
+  nullable    = false
+}
+
+variable "delivery_frequency" {
+  description = "The frequency with which AWS Config recurringly delivers configuration snapshots"
+  type        = string
+  default     = "Three_Hours"
+  nullable    = false
+}
+
+variable "include_global_resource_types" {
+  description = <<-EOF
+    Specifies whether AWS Config includes all supported types of global
+    resources with the resources that it records.
```

```suggestion
    Specifies whether AWS Config includes all supported types of global resources with the resources that it records.
```

Heredocs don't work like the YAML `>` operator and turn newlines into spaces, so reserve heredoc descriptions for actual multi-line strings and not wrapping lines.